runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -104,12 +105,18 @@ test {
 
 task validatesRunnerTest(type: Test) {
   group = "Verification"
+  dependsOn ":beam-runners-google-cloud-dataflow-java-legacy-worker:shadowJar"
   def dataflowProject = project.findProperty('dataflowProject') ?: 'apache-beam-testing'
   def dataflowTempRoot = project.findProperty('dataflowTempRoot') ?: 'gs://temp-storage-for-validates-runner-tests/'
+  def dataflowWorkerJar = project.findProperty('dataflowWorkerJar') ?: project(":beam-runners-google-cloud-dataflow-java-legacy-worker").shadowJar.archivePath
+
   systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
           "--runner=TestDataflowRunner",
           "--project=${dataflowProject}",
           "--tempRoot=${dataflowTempRoot}",
+          "--dataflowWorkerJar=${dataflowWorkerJar}",
+          "--workerHarnessContainerImage=",
```

Comment:
Why do we leave a blank workerHarnessContainerImage?

Reply:
To pick up the non-versioned container image, which handles a staged worker jar.

Reply:
This is only for the legacy worker and related to the Dataflow service. If the workerHarnessContainerImage isn't set to empty explicit, it will use the default one which contains the pre-build worker jar.

Reply:
Seems worthwhile for a comment since all other properties make sense since we are setting an explicit value.